### DISCUSSION TITLE
Are title bars implemented?

### DISCUSSION THREAD

**Discussion by copypasteonly** [leftwm/leftwm#993] [Q&A] [answered]
Hello, I was wondering if title bars are implemented in LeftWM? If so how do I enable them? I don't see them in my terminal and most of my apps don't have any title bar. Thanks!

**Comment by Eskaan** [comment, marked as answer]
I think leftwm windows (like most wms) don't have a title bar by default. To move/quit windows we use keybinds. If you are searching for a program's option menu, most of them open/close it when pressing the `alt` key (for example Firefox).

**Reply by copypasteonly** [reply]
Yeah I guess it really is not built in, I only wanted it for some aesthetics. Thanks!
